std/ws/mod.ts

```diff
@@ -31,6 +31,9 @@ export interface WebSocketCloseEvent {
   reason?: string;
 }
 
+/**
+   * Returns true if input value is a WebSocketCloseEvent, false otherwise.
+*/
```

Comment:
The formatting here is odd, and this could fit on a single line of text: https://deno.land/manual@v1.5.3/contributing/style_guide#use-jsdoc-for-exported-symbols

Reply:
How about right now?